.github/workflows/stale_labeler.yml

```
@@ -0,0 +1,20 @@
+  schedule:
+    - cron: "0 00 * * *"   # <=== Change this value
+
+jobs:
+  labeler:
+    name: Label Stale Things
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Run Srvaroa Labeler"
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)